src/main/java/net/minecraftforge/client/model/geometry/SimpleUnbakedGeometry.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.client.model.geometry;
+
+import com.mojang.datafixers.util.Pair;
+import net.minecraft.client.renderer.block.model.BakedQuad;
+import net.minecraft.client.renderer.block.model.ItemOverrides;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.client.resources.model.BakedModel;
+import net.minecraft.client.resources.model.Material;
+import net.minecraft.client.resources.model.ModelBakery;
+import net.minecraft.client.resources.model.ModelState;
+import net.minecraft.client.resources.model.UnbakedModel;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.client.RenderTypeGroup;
+import net.minecraftforge.client.model.IModelBuilder;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.function.Function;
+
+/**
+ * Base class for implementations of {@link IUnbakedGeometry} which do not wish to handle model creation themselves,
+ * instead supplying {@link BakedQuad baked quads} through a builder.
```

Comment:
This label should be in plain text, and therefore use `{@linkplain}`.
```suggestion
 * instead supplying {@linkplain BakedQuad baked quads} through a builder.
```